src/Microsoft.AspNetCore.Mvc.Abstractions/project.json

```diff
@@ -37,16 +37,15 @@
       "version": "1.1.0-*",
       "type": "build"
     },
-    "Microsoft.Net.Http.Headers": "1.1.0-*"
+    "Microsoft.Net.Http.Headers": "1.1.0-*",
+    "NETStandard.Library": "1.6.1-*"
   },
   "frameworks": {
     "net451": {},
     "netstandard1.3": {
       "dependencies": {
         "Microsoft.CSharp": "4.0.1-*",
```

Comment:
Err, missed this one. It's not in NETStandard.Library and should get bumped up to 4.3.0